Source/Initialization/PlasmaInjector.cpp

```diff
@@ -118,8 +118,13 @@ PlasmaInjector::PlasmaInjector (int ispecies, const std::string& name)
         mass = species::get_mass( physical_species );
     }
 
-    std::string s_inj_style;
-    pp_species_name.query("injection_style", s_inj_style);
+    // Parse injection style
+    std::string injection_style;
```

Comment:
Probably a good default:
```suggestion
    std::string injection_style = "none";
```

Reply:
Thank you. Done in https://github.com/ECP-WarpX/WarpX/pull/2081/commits/9066432544aac65d27504a6639e1388b3d8c9314.